<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1015</width>
    <height>737</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QRadioButton" name="polyShape">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>60</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Polygon</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>9</y>
      <width>101</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Annotation</string>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>0</width>
      <height>0</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>281</width>
      <height>301</height>
     </rect>
    </property>
    <property name="title">
     <string>Images</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>258</width>
       <height>254</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QListWidget" name="WidgetImgList"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Load Image</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="imgSortAsc">
          <property name="text">
           <string>Sort Asc</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="imgSortDesc">
          <property name="text">
           <string>Sort Desc</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>591</width>
      <height>641</height>
     </rect>
    </property>
    <property name="title">
     <string>Image loaded</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>571</width>
       <height>611</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QGraphicsView" name="graphicsView"/>
      </item>
      <item>
       <widget class="QSplitter" name="splitter_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QPushButton" name="pushButton_5">
         <property name="text">
          <string>Save Annotaions</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_6">
         <property name="text">
          <string>Load Annotaions</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>281</width>
      <height>291</height>
     </rect>
    </property>
    <property name="title">
     <string>Classes</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>261</width>
       <height>251</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QListWidget" name="classList"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="addClassText"/>
        </item>
        <item>
         <widget class="QPushButton" name="addClass">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="saveClass">
        <property name="text">
         <string>Save Class</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="loadClass">
          <property name="text">
           <string>Load Class</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deleteClass">
          <property name="text">
           <string>Delete Class</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
     <zorder>saveClass</zorder>
     <zorder>classList</zorder>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Search Image</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>261</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLineEdit" name="searchLineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="searchButton">
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>groupBox_5</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>splitter_5</zorder>
   <zorder>groupBox_4</zorder>
   <zorder>polyShape</zorder>
   <zorder>groupBox</zorder>
   <zorder>groupBox_3</zorder>
   <zorder>searchButton</zorder>
   <zorder>searchLineEdit</zorder>
   <zorder></zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1015</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionFAQ">
   <property name="text">
    <string>Contact</string>
   </property>
  </action>
  <action name="actionF_A_Q">
   <property name="text">
    <string>F.A.Q.</string>
   </property>
  </action>
  <action name="actionBug_Report">
   <property name="text">
    <string>Bug Report</string>
   </property>
  </action>
  <action name="actionSoftware_Information">
   <property name="text">
    <string>Software Information</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
